target/linux/x86/base-files/etc/board.d/02_network

```diff
@@ -31,6 +31,9 @@ cisco-mx100-hw)
 	ucidef_set_network_device_path "eth11" "pci0000:00/0000:00:01.1/0000:02:00.2"
 	ucidef_set_interfaces_lan_wan "mgmt eth2 eth3 eth4 eth5 eth6 eth7 eth8 eth9 eth10 eth11" "wan"
 	;;
+mellanox-technologies-ltd-msn2100)
+	ucidef_set_interface_lan "eth1 eth2 eth3 eth4 eth5 eth6 eth7 eth8 eth9 eth10 eth11 eth12 eth13 eth14 eth15 eth16"
```

Comment:
Please have a look how Realtek switches are handled [here](https://github.com/openwrt/openwrt/blob/main/target/linux/realtek/base-files/etc/board.d/02_network). Switches use `lan` rather than `eth` for interface names. Maybe @blogic or @dangowrt could comment on this.

Another idea could be to port the `lan_list` function to something more generic and use the variable instead of manually defining the interfaces.

Reply:
Nevermind this, keep it with eth.

Reply:
Mellanox goes with swp so maybe go to that?